### PR TITLE
Skip ci for nf core bot

### PR DESCRIPTION
**Pull request by jfy133**
As sugested on [s lack](https://nfcore.slack.com/archives/CE5LG7WMB/p1674652066038959)

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]

**Comment by mashehu** [comment]
hmm, one problem from this: if we skip the ci tests, the failed prettier run is also not rerun, no?

**Comment by maxulysse** [comment]
> hmm, one problem from this: if we skip the ci tests, the failed prettier run is also not rerun, no?

Good point

**Comment by maxulysse** [comment]
Can we add `[skip ci]` only if we say `@nf-core-bot fix linting [skip ci]`?

**Review by mashehu — CHANGES_REQUESTED** [review]
need a find a way that this only triggers the prettier ci

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@
 ### Template
 
 - Turn on automatic clean up of intermediate files in `work/` on successful pipeline completion in full-test config ([#2163](https://github.com/nf-core/tools/pull/2163))
+- Added `[skip ci]` to nf-core linting bot to stop executing entire pipeline CI tests for file formatting changes
```
Comment (jfy133) [inline CHANGELOG.md:8]:
```suggestion
- Added `[skip ci]` to nf-core linting bot to stop executing entire pipeline CI tests for file formatting changes ([2171](https://github.com/nf-core/tools/pull/2171)
)
```

**Comment by mashehu** [comment]
See https://github.com/nf-core/methylseq/pull/296 for exceptions

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,12 +4,14 @@
...
 - Turn on automatic clean up of intermediate files in `work/` on successful pipeline completion in full-test config ([#2163](https://github.com/nf-core/tools/pull/2163)) [Contributed by @jfy133]
 - Add documentation to `usage.md` on how to use `params.yml` files, based on nf-core/ampliseq text ([#2173](https://github.com/nf-core/tools/pull/2173/)) [Contributed by @jfy133, @d4straub]
 - Make jobs automatically resubmit for a much wider range of exit codes (now `104` and `130..145`) ([#2170](https://github.com/nf-core/tools/pull/2170))
 - Remove problematic sniffer code in samplesheet_check.py that could give false positive 'missing header' errors ([https://github.com/nf-core/tools/pull/2194]) [Contributed by @Midnighter, @jfy133]
 - Consistent syntax for branch checks in PRs ([#2202](https://github.com/nf-core/tools/issues/2202))
 - Fixed minor Jinja2 templating bug that caused the PR template to miss a newline
+- Added `[skip ci]` to nf-core linting bot to stop executing entire pipeline CI tests for file formatting changes ([2171](https://github.com/nf-core/tools/pull/2171)
```
Comment (ewels) [inline CHANGELOG.md:14]: uh... no you didn't? 😅

Comment (ewels) [reply CHANGELOG.md:14]: (interpreting `[skip ci]` in a fairly strict way as meaning in a commit message)

Comment (jfy133) [reply CHANGELOG.md:14]: It was before, and then we've dropped it - I've not yet updated the CHANGELOG as it seems we would be updating the system... (it was too simplistic the first time around)

**Comment by ewels** [comment]
ok, reading around a bit, apparently GitHub treats _skipped_ jobs as passing in terms of required tests. So if we get a workflow to run but skip the job, it might be fine...? As opposed to not triggering it and it not running at all. If that makes sense.

Needs a bit of experimentation I think.

**Pull request closed without merging**